azure-pipelines.yml

```diff
@@ -257,6 +257,7 @@ stages:
       - template: ci-templates/native-build-steps.yaml
         parameters:
           modules:
+            - cache
```

Comment:
You might have to tweak the timeout. I think the rule of thumb is 5 minutes per module.

Reply:
Timeout added.